src/plone/restapi/tests/test_types.py

```diff
@@ -311,6 +312,81 @@ def test_float(self):
             adapter.get_schema(),
         )
 
+    def test_email(self):
+        field = Email(
+            title=u"Email",
+            description=u"Email field",
+            default="foo@bar.com",
+            min_length=10,
```

Comment:
@avoinea if I am not mistaken min/max length does not really work in Volto, right? I mean if you set min/max length on the backend and Volto ignores it the REST API will raise an error if a user does not enter a text within a limit. Did you amend Volto to take min/max length into account on the frontend?

Reply:
@tisto Yes, my colleague @rexalex is working on it on https://github.com/plone/volto/issues/1246